### PR TITLE
User guide for SAAJ updated.

### PR DESCRIPTION
**Pull request by Tomas-Kraus**
Here is the guide: [SAAJ-TCK-Users-Guide.pdf](https://github.com/eclipse-ee4j/jakartaee-tck/files/4966360/SAAJ-TCK-Users-Guide.pdf)

### CODE REVIEW

File: install/saaj/docs/SAAJ2.0-ReleaseNotes.html [hunk 1]
```diff
@@ -77,24 +77,23 @@ <h2><a name="platform">Platform Notes</a></h2>
 <p>The Jakarta SOAP Attachments TCK tests run on the following platforms:</p>
 
 <ul>
-<li>Oracle Linux 7.1</li>
-<li>Windows 10</li>
+<li>CentOS 7</li>
```
Comment (scottmarlow) [inline install/saaj/docs/SAAJ2.0-ReleaseNotes.html:80]: Linux would be more generic than CentOS, could we use that?

File: user_guides/saaj/src/main/jbake/content/intro.adoc [hunk 2]
```diff
@@ -184,8 +184,7 @@ endif::end-to-end-tests[]
 
 The {TechnologyShortName} TCK tests run on the following platforms:
 
-* Microsoft Windows 10
-* Oracle Linux 7.1
+* CentOS 7
```
Comment (scottmarlow) [inline user_guides/saaj/src/main/jbake/content/intro.adoc:187]: Linux would be more generic than CentOS.

File: install/saaj/docs/SAAJ2.0-ReleaseNotes.html (hunk 1 above)
Comment (scottmarlow) [reply install/saaj/docs/SAAJ2.0-ReleaseNotes.html:80]: Other changes look fine to me.

Comment (Tomas-Kraus) [reply install/saaj/docs/SAAJ2.0-ReleaseNotes.html:80]: This was discussed on https://github.com/eclipse-ee4j/jakartaee-tck/pull/380 too. Think CentOS 7 is OK.

File: user_guides/saaj/src/main/jbake/content/intro.adoc (hunk 2 above)
Comment (Tomas-Kraus) [reply user_guides/saaj/src/main/jbake/content/intro.adoc:187]: Leaving CentOS 7 there.

**Comment by Tomas-Kraus** [comment]
Did local build. Bundle name is `soap-tck-2.0.0_31-Jul-2020.zip`. Removing date so there will be `soap-tck-2.0.0.zip`.
>     * [X]  attributes.conf: Update the following properties: {TechnologyRI} to EclipseGlassFish 6.0;  {TechnologyRIURL} to https://projects.eclipse.org/projects/ee4j.glassfish; {SpecificationInquiryList} to [jaxws-dev@eclipse.org](mailto:jaxws-dev@eclipse.org); {TCKPackageName} to jakarta-soap-tck-2.0.0.zip
> 
>     * [X]  req-software.inc - Change first example, "Java EE 8 RI" to "Eclipse GlassFish 6.0 Compatible Implementation"
> 
>     * [ ]  install-server.inc - please update with current terminology (Java EE 8 RI -> Jakarta EE 9 Compatible Platform (some examples: Eclipse GlassFish 6); change RI to Compatible Implementation; Change software can be obtained from, to point to the Jakarta EE 9 Platform Specification page, or suggest the user can obtain a compatible implementation from Eclipse GlassFish 6).
> 
>     * [ ]  debug-tips.inc - please delete the code header at the top of this file. Otherwise, this header is rendered in Chapt. 6, Debugging Test Problems (debug.adoc).
> 
>     * [ ]  rebuild.inc - Please review this file and update language, component and package names (RI to Compatible Implementation, Etc.). This is rendered in Appendix B (rebuild.adoc).
> 
>     * [ ]  install/saaj/docs/SAAJ2.0-ReleaseNotes.html: please update the release date in the page title.

File: user_guides/saaj/src/main/jbake/content/intro.adoc (hunk 2 above)
Comment (scottmarlow) [reply user_guides/saaj/src/main/jbake/content/intro.adoc:187]: :+1:

**Comment by edbratt** [comment]
The final build target names were established by the Specification Committee. You can use [this folder](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-900/) to find the final names of all TCKs generated by the release process by this project. I guess Local builds produce different file names. I wasn't aware of that.
Will you be able to check off the changes I asked for, or do I need to look for them individually in subsequent commits?

**Comment by Tomas-Kraus** [comment]
Thanks for the info. Name should be OK now.

File: user_guides/saaj/src/main/jbake/content/attributes.conf [hunk 3]
```diff
@@ -2,8 +2,8 @@
 :TechnologyShortName: SOAP Attachments
 :LegacyAcronym: SAAJ
 :TechnologyVersion: 2.0
-:TechnologyRI: GlassFish 6.0
-:TechnologyRIURL: https://eclipse-ee4j.github.io/glassfish/
+:TechnologyRI: EclipseGlassFish 6.0
```
[the remark this answers is not in the document]
Comment (Tomas-Kraus) [reply user_guides/saaj/src/main/jbake/content/attributes.conf]: OK, fixed.

**Comment by Tomas-Kraus** [comment]
I just need someone to merge it. I'm not a commiter.

**Pull request merged**